src/Tribe/Editor/Blocks/Event_Datetime.php

```diff
@@ -55,10 +55,12 @@ public function assets() {
 	 * Returns the block data for the block editor.
 	 *
 	 * @since 5.1.1
+	 * 
+	 * @param int $event_id The event post ID. Defaults to the current event.
 	 *
 	 * @return array<string,mixed> The block data for the editor.
 	 */
-	public function block_data() {
+	public function block_data( $event_id = null ) {
```

Comment:
This function is defined in the abstract class in common. If we update this here, we need to update this in the abstract class, meaning we'll need to update this for every block. We probably need to think of a different solution here.